containers/package-builder/build-concent-api-assets.sh

```diff
@@ -4,15 +4,16 @@ concent_dir=$(realpath "$1")
 package_builder_dir=$(realpath "$2")
 build_dir=$(realpath "$3")
 output_dir=$(realpath "$4")
-python_version=3.5
+python_version=3.6
 
 mkdir --parents "$build_dir/concent-api-assets/"
 cp --recursive "$concent_dir/concent_api" "$build_dir/concent-api-assets"
 
 cd "$build_dir/concent-api-assets/concent_api/"
 
-virtualenv --python python3 "$build_dir/virtualenv/"
+python$python_version -m venv "$build_dir/virtualenv/"
```

Comment:
Doesn't `python3-venv` provide the `virtualenv` command?

Reply:
No, It does't provide `virtualenv` command.